query-engine/connector-test-kit-rs/query-engine-tests/tests/queries/distinct.rs

```diff
@@ -57,7 +57,7 @@ mod distinct {
 
         assert_query!(
             runner,
-            "query { findManyUser(distinct: [first_name, last_name]) { id, first_name, last_name } }",
+            "query { findManyUser(distinct: [first_name, last_name], orderBy: {id: asc}) { id, first_name, last_name } }",
             r#"{"data":{"findManyUser":[{"id":1,"first_name":"Joe","last_name":"Doe"},{"id":2,"first_name":"Hans","last_name":"Wurst"}]}}"#
         );
```

Comment:
Yields:
SQL Error `SELECT DISTINCT ON expressions must match initial ORDER BY expressions`

Running the query _without_ the `orderby` yields a result set that is inverse to our current snapshot.
Expected:
```json
[
    { "id": 1, "first_name": "Joe", "last_name": "Doe" }, 
    { "id": 2, "first_name": "Hans", "last_name": "Wurst" },
]
```

Running _with_ the `orderby` forces whatever is in the `orderby` to _also_ be in the `distinct` set which then also affects the final result set:
```json
[
    { "id": 1, "first_name": "Joe", "last_name": "Doe" }, 
    { "id": 2, "first_name": "Hans", "last_name": "Wurst" },
    { "id": 3, "first_name": "Joe", "last_name": "Doe" }
]
```